install/jaxws/docs/JAXWS3.0-ReleaseNotes.html

```diff
@@ -152,31 +152,31 @@ <h2><a name="props"><code>jakarta.xml.ws.Endpoint</code> publishing API's</a></h
           publish the endpoints and an exception will be thrown. When this result occurs
           with this scenario, the result is considered a PASS.
         </p>
-        
+
         <p>
           The <code>http.server.supports.endpoint.publish.2</code> property settings are used to specify
           whether or not endpoint publishing is supported by the Jakarta EE 8 CI HTTP server.
           The same settings and caveats as the <code>http.server.supports.endpoint.publish
             </code> property apply to this property, based on whether or not the Jakarta EE 8 CI
           supports endpoint publishing.
         </p>
-        
+
         <hr>
         <h2><a name="install_setup_run">Installing, Setting Up, and Running the Jakarta XML Web Services TCK</a></h2>
-        
-        <p>Refer to the <cite>Jakarta XML Web Services TCK 2.3 User's Guide</cite>, available in the Jakarta XML Web Services TCK 2.3 documentation bundle, for complete instructions on installing, setting up, and running the Jakarta XML Web Services TCK.
+
+        <p>Refer to the <cite>Jakarta XML Web Services TCK 3.0 User's Guide</cite>, available in the Jakarta XML Web Services TCK 3.0 documentation bundle, for complete instructions on installing, setting up, and running the Jakarta XML Web Services TCK.
         The online version of the JT Harness version 5.0 documentation is available <a href="https://wiki.openjdk.java.net/display/CodeTools/Documentation">here</a>.</p>
-        
+
         <hr>
         <h2><a name="install_config_tomcat8">Quick Steps to Configure Apache Tomcat 8 for Jakarta XML Web Services TCK</a></h2>
-        
-        <p>Follow the steps below to configure Apache Tomcat 8 for Jakarta XML Web Services TCK 2.3</p>
-        
+
+        <p>Follow the steps below to configure Apache Tomcat 10 for Jakarta XML Web Services TCK 3.0</p>
+
         <ul>
-          <li>Install Apache Tomcat 8.5.x to /apache-tomcat-8.5.x</li>
-          <li>Install Jakarta XML Web Services RI 2.3 to /jaxws-ri</li>
-          <li>Install Jakarta XML Web Services TCK 2.3 to /jaxwstck</li>
-          <li>Copy Jakarta XML Web Services RI 2.3 and Jakarta XML Web Services TCK 2.3 jars to appropriate Apache Tomcat 8.5.x directories
+          <li>Install Apache Tomcat 8.5.x to /apache-tomcat-10.0.x</li>
```

Comment:
@Tomas-Kraus did you intend to update ```Apache Tomcat 8.5.x``` to ```Apache Tomcat 10.0.x```?  Currently, we are referring to both 8.5.x + 10.0.x.

Reply:
It must be 10.0. I was running JAX-WS TCK tests with tomcat while working on it with Lukas and GF was not yes available and we used 10.0.